test/libethereum/test/performance/rc5.sol

```diff
@@ -0,0 +1,101 @@
+// https://people.csail.mit.edu/rivest/Rivest-rc5rev.pdf
```

Comment:
How about that.  Fair use?  It's only performance test code, not even debugged yet.